tests/MoBi.Tests/Core/CheckNameVisitorSpecs.cs

```diff
@@ -505,21 +550,29 @@ public class When_renaming_an_application_builder : concern_for_CheckNameVisitor
       private IReadOnlyList<IStringChange> _resultChanges;
       private readonly string _oldName = "OldName";
       private ApplicationBuilder _applicationBuilder;
+      private EventGroupBuildingBlock _buildingBlock;
 
       protected override void Context()
       {
          base.Context();
-         _applicationBuilder = new ApplicationBuilder();
-         _applicationBuilder.SourceCriteria = Create.Criteria(x => x.With(_oldName));
-         var appTransport = new TransportBuilder();
-         appTransport.SourceCriteria = Create.Criteria(x => x.With(_oldName));
-         appTransport.TargetCriteria = Create.Criteria(x => x.With(_oldName));
+         _buildingBlock = new EventGroupBuildingBlock();
+         _applicationBuilder = new ApplicationBuilder
+         {
+            SourceCriteria = Create.Criteria(x => x.With(_oldName))
+         };
+         var appTransport = new TransportBuilder
+         {
+            SourceCriteria = Create.Criteria(x => x.With(_oldName)),
+            TargetCriteria = Create.Criteria(x => x.With(_oldName))
+         };
          _applicationBuilder.AddTransport(appTransport);
+
+         _buildingBlock.Add(_applicationBuilder);
       }
 
       protected override void Because()
       {
-         _resultChanges = sut.GetPossibleChangesFrom(A.Fake<IObjectBase>(), "NewName", _applicationBuilder, _oldName);
+         _resultChanges = sut.GetPossibleChangesFrom(_applicationBuilder, "NewName", _buildingBlock, _oldName);
```

Comment:
I have no idea why this worked here. The argument were even reversed!! with the object being renamed not being passed in the right place lol